file.go

```diff
@@ -0,0 +1,276 @@
+package ipam
+
+import (
+	"context"
+	"encoding/json"
+	"io/fs"
+	"os"
+	"path"
+	"sync"
+	"syscall"
+	"time"
+)
+
+type file struct {
+	path       string
+	prettyJSON bool
+	modTime    time.Time
+	parent     Storage
+	lock       sync.RWMutex
+}
+
+var nullModTime time.Time
+var DefaultLocalFilePath string
+
+func init() {
+	nullModTime = time.Unix(0, 0)
+	DefaultLocalFilePath = path.Join(getXDGDataHome(), "go-ipam", "ipam-db.json")
```

Comment:
```suggestion
	defaultLocalFilePath = path.Join(getXDGDataHome(), "go-ipam", "ipam-db.json")
```

Reply:
this is actually used as a default for CLI